lib/actions/wallet.js

```diff
@@ -220,6 +239,55 @@ function selectProposal(walletId, proposalId) {
   };
 }
 
+/**
+ * send a transaction
+ * @param {string} - walletId
+ * @param {object} - options
+ * @param {string} - options.passphrase
+ * @param {string} - options.value
+ * @param {string} - options.address
+ * @param {string} - options.accountId
+ * @dispatches
+ * @returns {void}
+ */
+function sendTX(walletId, options) {
+  return async dispatch => {
+    const { wallet } = getClient();
+    const { passphrase, value, address, accountId } = options;
+    let { rate } = options;
+
+    if (typeof rate === 'string') rate = parseInt(rate, 10);
+
+    assert(passphrase, 'must enter passphrase');
+    assert(rate, 'must enter rate');
+    assert(value, 'must enter value');
+    assert(address, 'must enter recipient address');
+
+    const opts = {
+      passphrase,
+      rate,
+      outputs: [{ value, address }],
+    };
+
+    try {
+      const response = await wallet.send(walletId, opts);
+      const txid = response.hash;
+      dispatch(setSendTXSuccess(walletId, accountId, txid, response));
+    } catch (e) {
+      dispatch(
+        setSendTXFailure(walletId, accountId, {
```

Comment:
Maybe address would also be useful to store here as well

Reply:
Yeah, I think so. Could be useful to debug why a transaction didn't go through